### ISSUE TITLE
Create an actual release

### ISSUE THREAD

**Issue by josegonzalez** [FriendsOfPHP/PHP-CS-Fixer#151]
Hi Fabien,

Love what you've done with the place. Is that a new binary you're releasing? It's pretty cool, and gets lots of usage.

In fact, there is even a [homebrew-php formula](https://github.com/josegonzalez/homebrew-php/blob/master/Formula/php-cs-fixer.rb) for it. Pretty cool (I don't use it, since all my code is shit anyhow and it would probably tell me to go jump in a very big ditch).

In any case, you don't appear to have made any releases, which means the sha changes all the time, and there isn't any proper upgrade path. Might I humbly suggest you do so, at least for my own sanity?

If you are hesitant to do so - because this is under heavy development or you hate life or because I am screaming in my avatar - then I would suggest just tagging releases starting with `1`, then proceeding to `2`, and so on and so forth. No, it's not semantic versioning, but you'll be pretty high up in releases by the end of this.

In summary, if you could make releases of this package, that would make my life easier ever so slightly and would be very much appreciated.

Yours in armed combat,
Jose
